psi4/src/psi4/libfock/SplitJK.h

```diff
@@ -137,6 +154,14 @@ class PSI_API SplitJK {
     virtual std::string get_COSX_grid() {
         throw PSIEXCEPTION("SplitJK::get_COSX_grid was called, but COSX is not being used!");
     };
+
+    virtual void set_snLinK_incfock_iter(bool incfock_iter) {
```

Comment:
What is the benefit of calling `snLinK` specific functions in the `SplitJK` base class? Wouldn't it make sense to include them in the `snLinK` class itself and perform a type cast if the K algo is `snLinK`

Reply:
The real answer is "it didn't cross my mind", but you're absolutely right that downcasting is a more elegant solution. I'll update this to use typecasting instead.

Reply:
All right, done (at least for snLinK)!